.gometalinter.json

```diff
@@ -3,6 +3,7 @@
     "gofmt",
     "goimports",
     "golint",
+    "ineffassign",
```

Comment:
I could not add `varcheck` here, because if I add this, the codecheck CI will fails a lot.
Maybe next time, we could consider to add it back to check var.